db/kafka/docker-compose.yml

```diff
@@ -10,8 +10,7 @@ services:
       - data
       - utility
       - ingest
-      - editor
-      - subscriber
+      - app
```

Comment:
Nginx needs both the Editor and Subscriber applications to be running, or it will crash.  So I've merged them.